modules/mvt/docs/api-reference/mapbox-vector-tile-loader.md

```diff
@@ -34,12 +34,22 @@ const loaderOptions = {
 const geoJSONfeatures = await load(url, MVTLoader, loaderOptions);
 ```
 
+## Outputs
+
+### GeoJSON-like with local coordinates
+
+The parser will return an array of GeoJSON-like objects with local coordinates in a range between `[0 - (bufferSize / tileExtent), 1 + (bufferSize / tileExtent)]` and feature properties from MVT by default.
```

Comment:
Yes, you are right. MVT can have a buffer for handling overlapping that is what would make local coordinates to be negative within a tile.

Those values are transparent for the application itself. It will just handle local coordinates without worrying about buffer size or extent. We are the ones converting the original tile extent to an extent of 1. But I wanted to tell users that there can be negative values or values greater than 1 because of the buffer cells within the tile.

Extent is part of [MVT tile specification](https://github.com/mapbox/vector-tile-spec/blob/master/2.1/vector_tile.proto#L70) and the buffer size is taken into account when generating the MVT tile, but not needed when decoding the tile.